src/__tests__/App.spec.js

```diff
@@ -29,7 +29,7 @@ describe('initial state', () => {
       </Provider>,
     );
 
-    expect(updateLocationHookSpy).toBeCalledTimes(1);
+    expect(updateLocationHookSpy).toBeCalled();
```

Comment:
This hook will correctly be called twice when we update index.js to use strict mode.  This test will be rewritten in the future to remove redux-mock-store and use a real redux store.